democracy/tests/test_comment.py

```diff
@@ -426,3 +314,41 @@ def test_get_plugin_data_for_comment(api_client, default_hearing):
         comment_list = get_data_from_response(api_client.get(url, {"include": "plugin_data"}))
         created_comment = [c for c in comment_list if c["id"] == response_data["id"]][0]
         assert created_comment["plugin_data"] == comment_data["plugin_data"][::-1]  # The TestPlugin reverses data
+
+
+@pytest.mark.parametrize('data', [{'section': 'nonexistingsection'}, None])
+@pytest.mark.django_db
+def test_post_to_root_endpoint_invalid_section(api_client, default_hearing, data):
+    url = '/v1/comment/'
+
+    response = api_client.post(url, data=data)
+    assert response.status_code == 400
+    assert 'section' in response.data
+
+    comment = default_hearing.sections.first().comments.first()
+    url = '%s%s/' % (url, comment.id)
+
+    response = api_client.put(url, data)
+    assert response.status_code == 400
+    assert 'section' in response.data
+
+
+@pytest.mark.django_db
+def test_root_endpoint_filters(api_client, default_hearing, random_hearing):
```

Comment:
Actually it isn't, it holds comments that should not be included if the filters work correctly.